### PR TITLE
feat(add-profile): web3community

### PR DESCRIPTION
**Pull request by krishguptadev**
Cc: @vinzvinci 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/704"></a>

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
LGTM! Is org. allow to add?

**Comment by krishguptadev** [comment]
> LGTM! Is org. allow to add?

Yep! I guess!

**Review by Panquesito7 — COMMENTED** [review]
I'm not sure if organizations are allowed to add 😅 @EddieHubCommunity/linkfree, thoughts?

**Comment by Panquesito7** [comment]
Actually, I think yes, it's allowed (see #703).

File: public/data/web3community.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Web3 Community",
+  "bio": "An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀",
```
Comment (Panquesito7) [inline public/data/web3community.json:3]:
```suggestion
  "bio": "An Open Source Community that focuses on decentralized applications, Web 3.0, and Blockchain technologies 🚀",
```

File: public/data/web3community.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "url": "https://web3community.github.io",
+      "icon": "globe"
+    },
+    {
+      "name": "Read our Blog",
+      "url": "https://dev.to/web3community",
+      "icon": "book"
+    },
+    {
+      "name": "Mail us",
```
Comment (Panquesito7) [inline public/data/web3community.json:17]: Optional, just an idea. 😃
```suggestion
      "name": "Contact us via mail",
```

File: public/data/web3community.json (hunk 1 above)
Comment (krishguptadev) [reply public/data/web3community.json:3]: It should lowercase as per grammarly.

Comment (Panquesito7) [reply public/data/web3community.json:3]: > It should lowercase as per grammarly.

Not for my Grammarly settings, though 👀 let's keep it as-is anyways.
But as you can see on the website we're using `Web 3.0`.

image

Comment (krishguptadev) [reply public/data/web3community.json:3]: As you can see in the github org bio, we are using web3 🙂

**Review by starlightknown — CHANGES_REQUESTED** [review]
Maybe make it follow us on LinkedIn than just LinkedIn organization? Otherwise it looks good to me

**Comment by krishguptadev** [comment]
> Maybe make it follow us on LinkedIn than just LinkedIn organization? Otherwise it looks good to me

Can we follow linkedin orgs? Sorry, I am not on LinkedIn, thus don't know.

**Comment by Vyvy-vi** [comment]
Maybe we should add a tag in the profile schema, that declares an org as an 'org'...

**Comment by krishguptadev** [comment]
> Maybe we should add a tag in the profile schema, that declares an org as an 'org'...

#696 🤔

**Comment by eddiejaoude** [comment]
I think they are a community and open source, then it could give the project some exposure too - this might change in the future, it is just my initial thinking

**Comment by emmalearnscode** [comment]
I think having some way to differentiate organisations from individuals would be a good idea. I might want to search just for orgs for example.

**Comment by Cahllagerfeld** [comment]
> I think they are a community and open source, then it could give the project some exposure too - this might change in the future, it is just my initial thinking

I think especially when the new partners program is ready. The partners should have benefits in comparision to non-partners I think. One might be to be listed in this project or something

**Comment by krishguptadev** [comment]
Maybe we shall refer #696 for this discussion 😅

**Review by Panquesito7 — APPROVED** [review]

**Comment by CodingSpecies** [comment]
> I think having some way to differentiate organisations from individuals would be a good idea. I might want to search just for orgs for example.

Yes, I agree with that @emmalearnscode .
I was thinking that there should be a clear difference between orgs and user profiles. Maybe at the end of each profile name, the emoji 📛, which is the badge could be put. `EG: Eddie Hub 📛`.
When you put 📛 in the search all the organisations will come up as they all have the same badge emoji in their name.

**Comment by eddiejaoude** [comment]
Great idea to differentiate 👍 , what about a field to do this - then we could have drop down or radio buttons to filter

**Comment by avneesh0612** [comment]
A dropdown would be cool 🤔👀

**Pull request merged**